test-common/uk/gov/hmrc/driverinspectionnotificationfrontend/helpers/AllMocks.scala

```diff
@@ -31,6 +33,7 @@ trait AllMocks extends BeforeAndAfterEach {
   val mockGmsService:             GmsService                                = mock[GmsService]
   val mockReferenceDataService:   GmsReferenceDataService                   = mock[GmsReferenceDataService]
 
+  @nowarn
```

Comment:
never used this annotation, can you supply the `value` to it to tell what warning we are suppressing ?
class nowarn(value: String = "")

rather thank introducing annotations, you can say `Seq[Any]` so the warning goes away which is much easier to follow going forward

Reply:
I prefer using `Seq[Any]` rather than using annotations on a unit test logic

Reply:
updated.